Javascript-2/Week-1/Movie App/arrayList.js

```diff
@@ -0,0 +1,460 @@
+export const movies = [
```

Comment:
The `export` keyword throws an error:
<img width="632" alt="image" src="https://github.com/Nusrath-Jahan/hyf-homework/assets/5467758/fe216069-499d-4c19-a619-53b34621bb0e">

If you want to use it, you must import the `arrayList.js` as a module in your HTML file